locales/fr/matches.json

```diff
@@ -0,0 +1,6 @@
+{
+    "econ": ["Communiste","Socialiste","Social","Centriste","Marché","Capitaliste","Laissez-Faire"],
```

Comment:
```suggestion
    "econ": ["Communiste","Socialiste","Social","Centriste","Marché","Capitaliste","Laisser-Faire"],
```

Reply:
Isn't either spelling acceptable when referring to the form of governance?

Reply:
You're right, `Laissez-Faire` seems to be the correct spelling in this context.